netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -253,10 +254,12 @@ typedef struct _net_ebpf_extension_connection_context
     uint16_t protocol;
     uint64_t timestamp;
     LIST_ENTRY list_entry;
+    uint32_t verdict;
```

Comment:
Why uint32_t here but int32_t in line 226?   Can they both be changed to say ebpf_sock_addr_verdict_t?

Reply:
Don't  typedef ebpf_sock_addr_verdict_t to int32_t, typedef it to an enum.